tests/python/TestVideos.py

```diff
@@ -0,0 +1,335 @@
+        addVideo["properties"] = props
+        addVideo["link"] = link
+
+        imgs_arr = []
+
+        fd = open("../test_videos/Megamind.avi", 'rb')
+        imgs_arr.append(fd.read())
+        fd.close()
+
+        img_params = {}
+
+        query = {}
+        query["AddVideo"] = addVideo
+
+        all_queries.append(query)
+
+        response, res_arr = db.query(all_queries, [imgs_arr])
+
+        self.assertEqual(response[0]["AddEntity"]["status"], 0)
+        self.assertEqual(response[1]["AddVideo"]["status"], 0)
+
+    def test_findVid_multiple_results(self):
+        db = vdms.vdms()
+        db.connect(hostname, port)
+
+        prefix_name = "vid_multiple"
+
+        number_of_inserts = 4
+        for i in range(0,number_of_inserts):
+            props = {}
+            props["name"] = prefix_name
+            self.insertVideo(db, props=props)
+
+        constraints = {}
+        constraints["name"] = ["==", prefix_name]
+
+        results = {}
+        results["list"] = ["name"]
+
+        img_params = {}
+        img_params["constraints"] = constraints
+
+        query = {}
+        query["FindVideo"] = img_params
+
+        all_queries = []
+        all_queries.append(query)
+
+        response, vid_arr = db.query(all_queries)
+
+        self.assertEqual(len(vid_arr), number_of_inserts)
+        self.assertEqual(response[0]["FindVideo"]["status"], 0)
+        self.assertEqual(response[0]["FindVideo"]["returned"], number_of_inserts)
+
+    def test_findVideoNoBlob(self):
+        db = vdms.vdms()
+        db.connect(hostname, port)
+
+        prefix_name = "fvid_no_blob_"
+
+        for i in range(0,2):
+            props = {}
+            props["name"] = prefix_name + str(i)
+            self.insertVideo(db, props=props)
+
+        all_queries = []
+
+        for i in range(0,2):
+            constraints = {}
+            constraints["name"] = ["==", prefix_name + str(i)]
+
+            results = {}
+            results["blob"] = False
+            results["list"] = ["name"]
+
+            img_params = {}
+            img_params["constraints"] = constraints
+            img_params["results"] = results
+
+            query = {}
+            query["FindVideo"] = img_params
+
+            all_queries.append(query)
+
+        response, img_array = db.query(all_queries)
+
+        self.assertEqual(response[0]["FindVideo"]["status"], 0)
+        self.assertEqual(response[1]["FindVideo"]["status"], 0)
+        self.assertEqual(len(img_array), 0)
+
+    def test_updateVideo(self):
+        db = vdms.vdms()
+        db.connect(hostname, port)
+
+        prefix_name = "fvid_update_"
+
+        for i in range(0,2):
+            props = {}
+            props["name"] = prefix_name + str(i)
+            self.insertVideo(db, props=props)
+
+        all_queries = []
+
+        constraints = {}
+        constraints["name"] = ["==", prefix_name + str(0)]
+
+        props = {}
+        props["name"] = "simg_update_0"
+
+        img_params = {}
```

Comment:
Do you need me to review tests? I am hoping you have put checks and balances here to make sure videos get stored and rertieved correctly without losing out data :)